### PR TITLE
Update addons-linter, amo-validator

### PR DESCRIPTION
**Pull request by wagnerand**

### CODE REVIEW

File: requirements/prod.txt [hunk 1]
```diff
@@ -15,10 +15,9 @@ PyJWT==1.4.0 \
     --hash=sha256:e1b2386cfad541445b1d43e480b02ca37ec57259fd1a23e79415b57ba5d8a694
 SQLAlchemy==0.7.5 \
     --hash=sha256:7e31190a15753694dcb6f4399ce7d02091b0bccf825272d6254e56144debfd18
-amo-validator==1.10.36 \
-    --hash=sha256:2c0eabc8d3ecd2bd2878e60704fa2dbeeaf86cda80f9feec74faad5b5e3e3fe0 \
-    --hash=sha256:749b163c825b2484c5b3c072bfb66ba41937b779636219bef11b2b118fcdb7e2
-# amqp is required by kombu
```
Comment (muffinresearch) [inline requirements/prod.txt]: Should this comment stay?

**Comment by muffinresearch** [comment]
r+ if travis is green.

**Comment by wagnerand** [comment]
Argh. I missed that hunk when interactively adding the file because the hashing script adds a lot more changes.

**Comment by eviljeff** [comment]
now there are more comments removed.  Is that correct?

**Comment by wagnerand** [comment]
Already fixed. What a mess.

**Pull request merged**